### PR TITLE
it is wednesday my dudes part 2

### PR DESCRIPTION
**Pull request by gobisa**
electric bugaloo

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -31,7 +32,8 @@ async def bing(ctx):
 
 @bot.command(name="day")
 async def day(ctx):
-    day = datetime.today().strftime("%A")
+    EST = timezone("US/Eastern")
```
Comment (BDokas) [inline src/bot.py:35]: but what about Swan, who doesn't live in eastern time zone?

Comment (dannyreilman) [inline src/bot.py:35]: Do we need to notify our dependencies of our new official timezone?

**Review by dannyreilman — CHANGES_REQUESTED** [review]

File: src/bot.py (hunk 1 above)
Comment (BDokas) [reply src/bot.py:35]: what about the shareholders overseas, wouldn't they like to use their local timezones too?

Comment (jswaneck) [reply src/bot.py:35]: :TEETH:

**Review by BDokas — APPROVED** [review]

**Review by dannyreilman — APPROVED** [review]

**Pull request merged**